bot/potentialScammer/getBotIntegralGuilds.js

```diff
@@ -0,0 +1,20 @@
+const getBotGuildsId = async (client) => {
+  //get the bot guilds id
```

Comment:
Only write comments when it add more meaning.
Here it just repeats the function name above. So it is redundant.